README.md

```diff
@@ -3315,51 +3319,33 @@ interface UserDataProviderInterface
 ```php
 <?php
 
+declare(strict_types=1);
+
 namespace Acme\NewsletterBundle\Entity;
 
 class UserData
 {
-    /**
-    * @var string
-    */
-    private $email;
-    
-    /**
-    * @var string
-    */
-    private $name;
+    private string $email;
+    private string $name;
 
-    /**
-     * @return string
-     */
-    public function getEmail()
+    public function getEmail(): string
```

Comment:
I am not sure if it's a correct sample... in general, I see that there is no constructor, which says that it's a must to add string, so this means... in case no one uses setEmail(), then it would return **: ?string**

Reply:
added constructor